doc/md_docs.go

```diff
@@ -105,9 +105,9 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 			link = strings.ReplaceAll(link, " ", "_")
 			buf.WriteString(fmt.Sprintf("* [%s](%s)\t - %s\n", cname, linkHandler(link), child.Short))
 		}
-		buf.WriteString("\n")
 	}
 	if !cmd.DisableAutoGenTag {
+		buf.WriteString("\n")
 		buf.WriteString("###### Auto generated by spf13/cobra on " + time.Now().Format("2-Jan-2006") + "\n")
```

Comment:
If we fix not having newlines at the end of the file, we should also remove the newline from the autogenerated tag at the end of the file:

```suggestion
		buf.WriteString("###### Auto generated by spf13/cobra on " + time.Now().Format("2-Jan-2006"))
```

Reply:
This fixes adding a second new line at the end of the file so this suggestion isn't needed